### PR TITLE
Creating title object before Ext does so we can customize values

### PR DESCRIPTION
**Pull request by eubmatt**
We wanted to set the default highlighted button to "No." Had to create this object before Ext did in order to change those values.

### CODE REVIEW

**Comment by RMcGhee** [comment]
LGTM

**Comment by ericpoe** [comment]
+1

File: classic/src/MessageBox.js [hunk 1]
```diff
@@ -25,6 +25,18 @@ Ext.define('Zan.common.MessageBox', {
             if ('yes' === buttonId) deferred.resolve(true);
         });
 
+        // This title object will be created anyways in the Msgbox.confirm() method if title is a string.
+        // We are creating it here so we can customize fields like the buttons and defaultFocus
+        title = {
```
Comment (JessicaJane) [inline classic/src/MessageBox.js:30]: I know the confirm method handles passing title as a config, but the documentation states it should be a string. 
Should we bypass the Ext.MessageBox.confirm method here and instead pass this config to Ext.MessageBox.show which is what confirm is doing?

**Comment by adeanzan** [comment]
Closing since this was implemented elsewhere

**Pull request closed without merging**